### PR TITLE
ZAP is no longer an OWASP project

### PR DESCRIPTION
**Pull request by kingthorin**
Left OWASP to SSP
- https://www.zaproxy.org/blog/2023-08-01-zap-is-joining-the-software-security-project/
- https://softwaresecurityproject.org/blog/welcoming-zap-to-the-software-security-project/

SSP fell through:
- https://www.zaproxy.org/blog/2024-03-13-zap-funding-and-the-open-source-fellowship/

Either way it's no longer an OWASP project.

### CODE REVIEW

File: OSTEscaner/OSTEscaner.py [hunk 1]
```diff
@@ -33,7 +33,7 @@ def check(self):
           output3 = subprocess.run("skipfish", shell=True, capture_output=True,text=True).stdout.strip("\n")
           if "skipfish web" in output3:
               self.skipfish=True         
-          return [[self.wapiti,"Wapiti {}".format(output[len(output)-5::])],[self.zap,"OWASP ZAP V2.10.0"],[self.nuclei,"Nuclei {}".format(output2[212::])],[self.nikto,"Nikto {}".format(output1[318:323])],[self.skipfish,"skipfish {}".format(output3[43:48])]]
```
Comment (kingthorin) [inline OSTEscaner/OSTEscaner.py:36]: There didn't seem to be a good reason to include a version here, especially an old one.

Comment (kingthorin) [reply OSTEscaner/OSTEscaner.py:36]: Does the formatting also need updates? I'm not sure if those are character ranges or something....

**Comment by kingthorin** [comment]
I don't want to change your project too much as I'm not familiar with it all, however you might want to change the app/project names to constants so that you don't have to change in so many places in the future.

**Review by OSTEsayed — APPROVED** [review]
Thank you, i didn't know that ZAP is no longer OWASP project.

**Pull request merged**